server/tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "experimentalDecorators": true,
     "allowSyntheticDefaultImports": true,
     "resolveJsonModule": true,
-    "target": "es2017",
+    "target": "es2021",
```

Comment:
Why did you change this? Does the type not exist in es2017 or something?

Reply:
Correct, it doesn't exist until 2021.